content/wireless.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: Wireless Troubleshooting
+title: Wireless and Basic Troubleshooting
```

Comment:
This new name doesn't make sense to me. "Basic Wireless Troubleshooting" would make more grammatical sense than "Wireless and Basic Troubleshooting." But does adding the word "basic" add any value or make this article easier for people to find?